src/gentropy/susie_finemapper.py

```diff
@@ -0,0 +1,140 @@
+"""Step to run a finemapping using."""
+
+from __future__ import annotations
+
+from typing import Any
+
+import numpy as np
+import pyspark.sql.functions as f
+from pyspark.sql import DataFrame, Window
+
+from gentropy.common.session import Session
+from gentropy.dataset.study_locus import StudyLocus
+
+
+class SusieFineMapperStep:
+    """SuSie finemaping. It has generic methods to run SuSie fine mapping for a study locus.
+
+    This class/step is the temporary solution of the fine-mapping warpper for the development purposes.
+    In the future this step will be refactored and moved to the methods module.
+    """
+
+    @staticmethod
+    def susie_inf_to_studylocus(
+        susie_output: dict[str, Any],
+        session: Session,
+        _studyId: str,
+        _region: str,
+        _join: DataFrame,
+        cs_lbf_thr: float = 2,
+    ) -> StudyLocus:
+        """Convert SuSiE-inf output to studyLocus DataFrame.
+
+        Args:
+            susie_output (dict[str, Any]): SuSiE-inf output dictionary
+            session (Session): Spark session
+            _studyId (str): study ID
+            _region (str): region
```

Comment:
Do we have a consisted way of representing regions? If so, in the args description could be written eg.:
```text
_region (str): finemapped region in chr:start-end format
```

Reply:
No, we don't have it now. But agree, we need to think about the standard.